src/core/currency/wallet/currency-wallet-callbacks.ts

```diff
@@ -348,8 +348,8 @@ export function makeCurrencyWalletCallbacks(
           files[txidHash],
           tx.tokenId
         )
-        if (isNew) created.push(combinedTx)
-        else if (files[txidHash] != null) changed.push(combinedTx)
+        // if (isNew) created.push(combinedTx)
+        // else if (files[txidHash] != null) changed.push(combinedTx)
```

Comment:
You probably want to do `changed.push(combinedTx)` for every transaction, so the GUI still gets updates.

The idea here is to switch between `onTxChanged` / `onNewTx`, but since we don't have files, we just want everything to go in the "existing" bucket. The `if (files[txidHash] != null)` check was there to prevent crashes, but that's no longer needed thanks to your other changes.